media-libs/mesa/mesa-9999.ebuild

```diff
@@ -316,6 +316,12 @@ pkg_setup() {
 	python-any-r1_pkg_setup
 }
 
+src_prepare() {
+	# Temporary workaround: https://gitlab.freedesktop.org/mesa/mesa/-/issues/7717#note_1832122
+	use rusticl && eapply "${FILESDIR}/clang_resource_dir.patch"
```

Comment:
Is there any danger in applying this unconditionally?

Reply:
I don't think so.